src/runtime/composables/content.ts

```diff
@@ -71,12 +71,12 @@ export const useContent = () => {
   /**
    * Next page from `surround`.
    */
-  const next = computed(() => surround.value?.[1])
+  const next = computed(() => surround?.value[1])
```

Comment:
```suggestion
  const next = computed(() => surround.value?.[1])
```
Surround is a ref, and we should check its value for `undefined`